### ISSUE TITLE
author and work missing in usage example for audi

### ISSUE THREAD

**Issue by monzug** [alpheios-project/alpheios_nemo_ui#297]
Lookup for audi then go to usage word examples. the first line does not have the work and author's name, only the line number. it should be Pseudo-Varro, Sententiae + line number.
obviously it cannot be searched by author (and work).

**Comment by balmas** [comment]
@monzug I can't reproduce this one.

**Comment by monzug** [comment]
![audi](https://user-images.githubusercontent.com/41396793/64636486-d1057e80-d3cf-11e9-9429-e079f1e5d3c3.JPG)

**Comment by monzug** [comment]
tested in About Alpheios
Alpheios Embedded Library 3.0.0-beta.1.38
Alpheios Components 1.2.38

**Comment by monzug** [comment]
interesting not reproducible in web ext 3.0.0.64

**Comment by balmas** [comment]
I can't reproduce it at the texts environment either. Very strange. I am guessing it's an error from the server.

**Comment by monzug** [comment]
oh, interesting. well, will install new build tomorrow and check again.

**Comment by monzug** [comment]
![IMG_3982](https://user-images.githubusercontent.com/41396793/64637798-8a655380-d3d2-11e9-9add-eb98aa32ec24.PNG)

**Comment by monzug** [comment]
mobile and text env shows the 127.1 link. but I am still on Alpheios Embedded Library 3.0.0-beta.1.38
Alpheios Components 1.2.38
on desktop,  Alpheios Reading Tools 3.0.0.65
Alpheios Components 1.2.39   is not reproducible

**Comment by balmas** [comment]
hmm.  I wonder if this could be caused by a bug with the inflection tables fix for alpheios-project/components#784 that @kirlat found and fixed in components 1.2.39.  There were some cases where the code was throwing an error. It shouldn't have affected the usage examples view, maybe that error interrupted the processing of the vue components.

**Comment by monzug** [comment]
looks more of data issue to me. when can I get a texts env with 1.2.39?

**Comment by balmas** [comment]
in a few minutes :-)

**Comment by monzug** [comment]
in safari build

**Comment by balmas** [comment]
it's there now.

**Comment by monzug** [comment]
tested in Alpheios Components 1.2.39 - still getting the same line link for
audi (127.1)

**Comment by balmas** [comment]
That is super weird. Can you do me a favor? BEFORE clicking on the usage examples button open your Chrome developer toolbar. Click on the button and then go to the Network tab of the developer toolbar. 
Find the request that looks like "audio?authMax=3&max=10000000 and then click the Response tab and copy the content of the response (by right clicking on it) and save it in a text file and send it to me.  (See attached screenshot which tries to show what I mean)
![Screenshot from 2019-09-11 08-40-41](https://user-images.githubusercontent.com/644870/64698080-301ad000-d470-11e9-90e8-d64dce1730b1.png)

**Comment by monzug** [comment]
ok

**Comment by balmas** [comment]
hmmm. just seeing this for another word, veni, whose first result is 59.1 and in the data data citation is [Var].Sent.59.1 loc/1041/1/0/4453-4457

**Comment by balmas** [comment]
Yes, we don't always get the same results for each other back from the server, that does appear to be somewhat random.  

In the case I see with veni, it appears that the server is actually returning an incorrect citation. the data in the cit field is [Var].Sent.59.1 but the link actually goes to [PsVar].Sent.59.1 (Pseudo-Varro instead of Varro). Since the example with audi is also the same author I think it's probably just another example of the same data error. We aren't showing anything because we aren't finding the author and title reference in our data file, and don't know how to translate them.  

The best I can do to fix this is either
(1) to show the citation abbreviation exactly as it is reported in the data (even though it's wrong)
(2) to drop the example entirely

Which do you think is better? And do you think it can wait for an incremental?

**Comment by monzug** [comment]
drop the example. and it can definitely wait, right?

**Comment by balmas** [comment]
agree.
